docs/configure/cli/automation-rules.md

```diff
@@ -88,3 +88,50 @@ The `--delete` flag is not available for the `push all` command to avoid unexpec
 For additional information see
 * [OPA policies With CLI](/configure/cli/opa)
 * [Policies Configuration with CLI](/configure/cli/settings)
+
+## Testing Automation Rules before inserting into Insights
+Before pushing Automation Rules to Insights, you can use the CLI to check if the results will be as expected:
```

Comment:
Before pushing Automation Rules to Insights, you can use the CLI to check if the results will work as expected: